### DISCUSSION TITLE
Mouse doesn't work on Android version

### DISCUSSION THREAD

**Discussion by jaherron** [ppy/osu#18064] [Q&A]
I have a Samsung Galaxy Tab S7+, so I wanted to try playing osu with my tablet's keyboard cover and a Logitech MX Master 2S mouse over Bluetooth. Sadly when I connect the mouse to my tablet over bluetooth, the mouse doesn't work.

**Comment by peppy** [comment]
Doesn't work at all (ie. no cursor movement)? If that's the case, it's likely a misconfigured bluetooth setup.

If you're having a more specific issue, please provide more details. But do note that mobile support is an ongoing process and we don't have any advanced mouse support on android as far as I'm aware, so it would just use whatever the OS is providing by default.

**Reply by jaherron** [reply]
The cursor moves and can do stuff outside the game, but can't be used within the game. The game only responds to fingers or the S-Pen.

**Comment by Susko3** [comment]
Mouse support for Android was recently improved in https://github.com/ppy/osu/pull/16464. Try it again and report back if it works. (You may need to tweak the in-game mouse settings to make it work.)
